src/Calculator/Controls/OverflowTextBlock.cpp

```diff
@@ -34,7 +34,7 @@ void OverflowTextBlock::OnApplyTemplate()
 
     m_expressionContainer = safe_cast<ScrollViewer^>(GetTemplateChild("expressionContainer"));
     m_expressionContainer->ChangeView(m_expressionContainer->ExtentWidth - m_expressionContainer->ViewportWidth, nullptr, nullptr);
-
+    m_expressionContainer->ViewChanged += ref new Windows::Foundation::EventHandler<Windows::UI::Xaml::Controls::ScrollViewerViewChangedEventArgs ^>(this, &CalculatorApp::Controls::OverflowTextBlock::OnViewChanged);
```

Comment:
Yes. I will also unregister the other events (`m_scrollRight->Click` and `m_scrollLeft->Click`) in case  `OnApplyTemplate` is called more than 1 time (it's not the case right now, but better to manage this scenario too)